qutebrowser/mainwindow/statusbar/progress.py

```diff
@@ -40,6 +40,9 @@ class Progress(QProgressBar):
             border: 2px solid transparent;
             margin-left: 1px;
             background-color: transparent;
+            font-size: 4px; /* force very small s.t. it does not affect size hint */
+                            /* if you choose the value too small, it does not */
+                            /* appear when restoring the session */
```

Comment:
Looking at the `sizeHint`, I'm not sure if it even was intended to use `self.fontMetrics()` or if that's just a wrong copy-paste.

What happens if you remove this and remove the `sizeHint`? What if you just return some sensible fixed value like `QSize(90, 4)` or so in `sizeHint`?